proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -82,6 +82,15 @@ service Query {
   option (google.api.http).get =
       "/interchain_security/ccv/provider/proposed_consumer_chains";
       }
+
+  // QueryAllPairsValConAddrByConsumerChainID returns a list of pair valconsensus address
+  // between provider and consumer chain
+  rpc QueryAllPairsValConAddrByConsumerChainID (
```

Comment:
Consider documenting this function under KeyAssignment in the docs.

https://github.com/cosmos/interchain-security/blob/main/docs/docs/features/key-assignment.md

Reply:
@ThanhNhann Any news on the docs?

Reply:
Will update it